### PR TITLE
Use simctl openUrl for 12.2 sims safari tests

### PR DESCRIPTION
**Pull request by imurchie**
Rather than use our web technology to open the URL, use simctl so it can happen before we listen to the Web Inspector.

### CODE REVIEW

File: lib/driver.js [hunk 1]
```diff
@@ -562,10 +562,13 @@ class XCUITestDriver extends BaseDriver {
...
       }
 
-      // TODO: platformVersion should be a required cap
-      if (this.isSafari() && util.compareVersions(this.opts.platformVersion, '>=', '12.2')) {
-        // on 12.2 the page is not opened in WDA
-        await this.setUrl(this._currentUrl);
+      if (this.isSafari() && !this.isRealDevice()) {
+        // on real devices this often causes other things to open
+        // TODO: platformVersion should be a required cap
+        if (util.compareVersions(this.opts.platformVersion, '>=', '12.2')) {
```
Comment (mykola-mokhnach) [inline lib/driver.js]: there's no need for nested if. We could combine the condition with `&&`

File: lib/driver.js [hunk 2]
```diff
@@ -562,10 +562,13 @@ class XCUITestDriver extends BaseDriver {
         await markSystemFilesForCleanup(this.wda);
       }
 
-      // TODO: platformVersion should be a required cap
-      if (this.isSafari() && util.compareVersions(this.opts.platformVersion, '>=', '12.2')) {
-        // on 12.2 the page is not opened in WDA
-        await this.setUrl(this._currentUrl);
```
Comment (mykola-mokhnach) [inline lib/driver.js]: should we keep this for real devices?

File: lib/driver.js (hunk 1 above)
Comment (imurchie) [reply lib/driver.js]: I'd rather nest when conditions are entirely different.

File: lib/driver.js (hunk 2 above)
Comment (imurchie) [reply lib/driver.js]: It can't happen here. Until we're in a web context this will just open Siri on a real device.

Comment (mykola-mokhnach) [reply lib/driver.js]: can we then set the url using normal xctest primitives (aka type it in the address bar and click Go?)

**Pull request merged**